src/main/java/frc/robot/utils/utils/Utils.java

```diff
@@ -0,0 +1,87 @@
+package frc.robot.utils.utils;
+
+public class Utils {
+    /**
+     * Gets the minimal error between two desired angles.
+     *
+     * @param targetAngle  the desired angle. [rad]
+     * @param currentAngle current angle. [rad]
+     * @return the distance between the angles. [rad]
+     */
+    public static double getTargetError(double targetAngle, double currentAngle) {
+        double cwDistance = targetAngle - currentAngle;
```

Comment:
I don't what direction you use, but usually, rotations' direction is CCW. So it seems more natural to me that the way from the current angle to the current angle will be CCW and not CW.